j/jetty/jetty-test-helper_5.3_ubi_8.4.sh

```diff
@@ -0,0 +1,61 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : jetty-test-helper
+# Version               : jetty-test-helper-5.3
+# Source repo           : https://github.com/eclipse/jetty.toolchain
+# Tested on             : UBI 8.4
+# Language              : Java
+# Travis-Check          : False
```

Comment:
@kumarvikas2020 can you clarify why this is set to false?

Reply:
This is corrected.